### DISCUSSION TITLE
no TABS in the source code

### DISCUSSION THREAD

**Discussion by ehan08** [OpenRadioss/OpenRadioss#427] [Contributors]
The coding guidelines suggest not to use TABS in the source code as a good coding style. 

It seems that some source files contain TABS. Some editors with different tab settings do not show them correctly. Is it possible to have a tool to scan this in future?

Thanks.

Edward

**Comment by MarianBulla** [comment]
Yes, I fully agree with you and thank you very much for this catch.
We will pay much more attention on that, now!
Yes, maybe we can create a small tool to scan for this issue.

Thanks and best regards,
Marian

**Comment by laurent-altr** [comment]
To find tabulation in Fortran files, you can run:
`find ./ -name "*.F" |xargs grep $'\t'`

**Reply by richb2k** [reply]
Running the above find command returns just under 20,000 hits.

**Reply by richb2k** [reply]
Replacing tabs with spaces makes sense.  What about trimming trailing white space from the source files?   Is this also a recommended coding style for OpenRadioss?

Rich.

**Comment by ehan08** [comment]
Yeah, it is nice to trim tailing white space/tabs.
